eberron.md

```diff
@@ -400,6 +409,8 @@ While in the berserker form:
 - You cannot cast spells
 - Re-roll 1 wound die for each of your attacks
 
+You start with 1 proficiency in Athletics and Intimidation, and 2 proficiency in a weapon type of your choice. You have a weapon matching your chosen proficiency.
```

Comment:
Backgrounds also determine your what gear (if any) you start with and your proficiencies. Adding something along the lines of what your going with. Feel free to change though.